src/main/java/ch/njol/skript/variables/VariableStorage.java

```diff
@@ -172,12 +215,13 @@ protected <T> T getValue(SectionNode sectionNode, String key, Class<T> type) {
 
 	/**
 	 * Loads the configuration for this variable storage
-	 * from the given section node.
+	 * from the given section node. Loads internal required values first in load_i.
+	 * {@link #load(SectionNode)} is for extending classes.
 	 *
 	 * @param sectionNode the section node.
 	 * @return whether the loading succeeded.
 	 */
-	public final boolean load(SectionNode sectionNode) {
+	public final boolean load_i(SectionNode sectionNode) {
```

Comment:
Not sure about a public method being suffixed `_i`. Could we come up with a different name?